app/components/article-media-gallery.js

```diff
@@ -37,9 +37,7 @@ export default Component.extend(
 		 * @returns {void}
 		 */
 		didRender() {
-			this.$().on('scroll', () => {
-				debounce(this, 'onScroll', 100);
-			});
+			this.element.addEventListener('scroll', this.debouncedScroll);
```

Comment:
We'll lose `this` here. You should bind `debouncedScroll` to `this` (`this.debouncedScroll  = this. debouncedScroll.bind(this)`)